src/index.js

```diff
@@ -45,6 +45,37 @@ const FSPIOP_DestinationCurrencyHeader = 'FSPIOP-DestinationCurrency'.toLowerCas
 const FSPIOP_SourceHeader = 'FSPIOP-Source'.toLowerCase();
 const FSPIOP_DestinationHeader = 'FSPIOP-Destination'.toLowerCase();
 
+const db = require('./events/persistence/db/database');
```

Comment:
How about moving this into a file that is `require`'d and executing the function from there?
Instead of adding more code to the index.js
On the new repo we also don't use the big index.js.

Reply:
This should be wrapped in a if condition to validate that `config.fxpModeEnabled === true`